composer.json

```diff
@@ -18,6 +18,8 @@
         "dmkit/phalcon-jwt-auth": "dev-master",
         "duncan3dc/fork-helper": "^2.2.0",
         "firebase/php-jwt": "^5.0",
+        "hybridauth/hybridauth": "dev-master",
```

Comment:
Do not use dev-master. Never. @rwhite27
If someone introduces a breaking change, you fucked.

Reply:
We are going to user 'dev-master' for now. Cant seem to find an stable version for version 3.

Reply:
What about using a pre-release tag?
https://github.com/hybridauth/hybridauth/tree/v3.0.0-rc.10
It is better than nothing.
This is the rc @rwhite27